meteor/client/ui/Prompter/PrompterView.tsx

```diff
@@ -881,9 +887,10 @@ export const Prompter = translateWithTracker<PropsWithChildren<IPrompterProps>,
 				segment.parts.forEach((part) => {
 					lines.push(
 						<div
-							id={`partInstance_${part.id}`}
+							id={`part_${part.id}`}
```

Comment:
Note to reviewer: this is the important bit. This will make the prompter treat PartInstances (temporary or not) of the same Part _as the same Part_, instead of an unrelated third Part.